File: CS/MVVMExpenses/Models/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace MVVMExpenses.DataModels {
    public class Account {
        [Key, Display(AutoGenerateField = false)]
        public long ID { get; set; }
        [Required, StringLength(30, MinimumLength = 4)]
        [Display(Name = "ACCOUNT")]
        public string Name { get; set; }
        [DataType(DataType.Currency)]
        [Display(Name = "AMOUNT")]
        public decimal Amount { get; set; }
        public override string ToString() {
            return Name + " (" + Amount.ToString("C") + ")";
        }
    }
}

File: VB/MVVMExpenses/Models/Account.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.ComponentModel.DataAnnotations

Namespace MVVMExpenses.DataModels
    Public Class Account
        <Key, Display(AutoGenerateField:=False)>
        Public Property ID() As Long
        <Required, StringLength(30, MinimumLength:=4), Display(Name:="ACCOUNT")>
        Public Property Name() As String
        <DataType(DataType.Currency), Display(Name:="AMOUNT")>
        Public Property Amount() As Decimal
        Public Overrides Function ToString() As String
            Return Name & " (" & Amount.ToString("C") & ")"
        End Function
    End Class
End Namespace